sdk/storage/storage-file/samples/typescript/advanced.ts

```diff
@@ -53,24 +53,24 @@ async function main() {
 
   // Parallel uploading with uploadFileToAzureFile in Node.js runtime
   // uploadFileToAzureFile is only available in Node.js
-  await uploadFileToAzureFile(Aborter.none, localFilePath, fileClient, {
+  await uploadFileToAzureFile(localFilePath, fileClient, {
     rangeSize: 4 * 1024 * 1024, // 4MB range size
     parallelism: 20, // 20 concurrency
-    progress: ev => console.log(ev)
+    progress: (ev) => console.log(ev)
```

Comment:
If you run `rushx format` will the parenthesis still be there? we should be using consistent formatting/linting rules.

Reply:
Update - Waiting for Eng Sys on `eslint`